libnymea-core/ruleengine/ruleengine.cpp

```diff
@@ -626,8 +632,9 @@ RuleEngine::RuleError RuleEngine::executeActions(const RuleId &ruleId)
     }
 
     qCDebug(dcRuleEngine) << "Executing rule actions of rule" << rule.name() << rule.id().toString();
-    m_logEngine->logRuleActionsExecuted(rule);
-    executeRuleActions(rule.actions());
+    m_logger->log({rule.id().toString(), "executed"}, {{"name", rule.name()}});
+//    m_logEngine->logRuleActionsExecuted(rule);
```

Comment:
will be logged by the thingmanager?

Reply:
all the actions executed by the rule will be logged later when the action finishes, so it can also log the status of the execution.

I've cleaned up the commented out line(s) here though. thanks